.gitignore

```diff
@@ -0,0 +1 @@
+**/*.DS_Store
```

Comment:
EOF

Reply:
`.DS_Store` is a kind of __'junk file'__ on MacOS, it generates out of developers' control...and, [cryptape/cita](https://github.com/cryptape/cita/blob/bcc0843e7011a03de2de289f6952c8826c1959aa/.gitignore#L33) also add `.DS_Store` into `.gitignore`, here is the [explanation](https://stackoverflow.com/questions/30246832/is-ds-store-file-important).

Reply:
yeah, i mean add a line at the end of file.
check the tips of github:

> No new line at end of file.

https://en.wikipedia.org/wiki/End-of-file

Reply:
check this:

https://github.com/cryptape/cita/blob/develop/.editorconfig#L9

Reply:
deal, `force-push` or `squash-merge`...